### PR TITLE
Integrate set availability in onboarding 

### PR DESCRIPTION
**Pull request by alvyjudy**
- This PR let the onboarding component post the available time and day to the database
- Days are now stored in an array whose number denotes day of the week. `0` is Sunday, `6` is Saturday.
- Solves the warning that says "children props received null" in `SetTimezoneUrl`

I will open another PR to actually use these availability data as that require more work than I anticipated.

### CODE REVIEW

**Review by kozaktar — CHANGES_REQUESTED** [review]
Can you also remove the edit buttons in onboarding dialog since they don't do anything?

**Comment by alvyjudy** [comment]
Good point Taras!

**Review by kozaktar — COMMENTED** [review]
Looks good Di!

**Review by kozaktar — APPROVED** [review]

File: server/db/models/User.js [hunk 1]
```diff
@@ -29,6 +29,12 @@ const schema = new Schema({
   URL: {
     type: String,
   },
+  availableTime: {
+    type: mongoose.Schema.Types.Mixed, // {start: "HH:MM", end: "HH:MM"}
```
Comment (dproc96) [inline server/db/models/User.js:33]: You could also define this as such:
```javascript
availableTime: {
  start: String,
  end: String
}
```

**Review by dproc96 — APPROVED** [review]
Other than my very small suggestion, this looks good to go!

**Comment by alvyjudy** [comment]
@WhoaNellie I have made the edits to store timezone as integer! Please take a look at line 26 on `OnBoarding.js` and line 147 on `SetTimezoneUrl.js`

**Pull request merged**